### ISSUE TITLE
CNN architecture invalid

### ISSUE THREAD

**Issue by MrEliptik** [MrEliptik/HandPose#5]
As @vincelui21 pointed out to me by email, the default padding value is 'none' so the output size should decrease to 26x26 and not 28x28. This propagates to the other conv layers as well. 
See keras.summary() output below 

`>>> model.summary()
_________________________________________________________________
Layer (type)                 Output Shape              Param #
=================================================
conv2d_1 (Conv2D)            (None, 26, 26, 32)        320
_________________________________________________________________
conv2d_2 (Conv2D)            (None, 24, 24, 64)        18496
_________________________________________________________________
max_pooling2d_1 (MaxPooling2 (None, 12, 12, 64)        0
_________________________________________________________________
dropout_1 (Dropout)          (None, 12, 12, 64)        0
_________________________________________________________________
flatten_1 (Flatten)          (None, 9216)              0
_________________________________________________________________
dense_1 (Dense)              (None, 128)               1179776
_________________________________________________________________
dropout_2 (Dropout)          (None, 128)               0
_________________________________________________________________
dense_2 (Dense)              (None, 6)                 774
================================================
Total params: 1,199,366
Trainable params: 1,199,366
Non-trainable params: 0
_________________________________________________________________`

CNN architecture image should be fixed.
